cg/utils/files.py

```diff
@@ -61,20 +61,20 @@ def is_pattern_in_file_path_name(file_path: Path, pattern: str) -> bool:
     return pattern in file_path.name
 
 
-def get_directory_creation_time_stamp(directory_path: Path) -> float:
+def get_source_creation_time_stamp(source_path: Path) -> float:
```

Comment:
this can be used on both dirs and files - made the name more general

Reply:
create a test for this function using a file